### PR TITLE
Add regression script for dpdk backend

### PR DESCRIPTION
**Pull request by hanw**
Reusing the p4test script with minor modification

### CODE REVIEW

File: backends/dpdk/run-dpdk-test.py [hunk 1]
```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python3
```
Comment (mbudiu-vmw) [inline backends/dpdk/run-dpdk-test.py:1]: I suspect much of this file is borrowed from the other run-* scripts. Ideally we should factor out the common parts into a module and put it in tools.

**Review by mbudiu-vmw — APPROVED** [review]

**Comment by mbudiu-vmw** [comment]
It looks like you have a syntax error in your CMakefile

File: backends/dpdk/run-dpdk-test.py (hunk 1 above)
Comment (fruffy) [reply backends/dpdk/run-dpdk-test.py:1]: I started working on a template in the eBPF back end with run-ebpf-test.py, but never got around to generalizing it to the other back ends. Maybe one day I will get around to porting it.

Comment (hanw) [reply backends/dpdk/run-dpdk-test.py:1]: @fruffy thanks, let me know when the template is ready.

**Pull request merged**